queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name like '%mon';
+SELECT * from animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT * from animals WHERE neutered AND escape_attempts <3;
+SELECT * from animals WHERE name IN ('Agumon', 'Pikachu');
+SELECT (name, escape_attempts) from animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE name NOT IN ('Gabumon');
+SELECT * from animals WHERE neutered;
+SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
```

Comment:
- [ ] **Good job so far 👏 👏** However, it was suggested in the project requirements to add a screenshot of all query results to the pull request description. 😂 Kindly, add them as the result of all your queries 👍
 
![image](https://user-images.githubusercontent.com/102757126/228016959-bf07ba38-a9b5-4547-9e48-4390f1638a83.png)

![image](https://user-images.githubusercontent.com/102757126/228461219-eaa8848c-b693-4f45-a9fd-53844d351cd2.png)